src/ResultsActivity/ResultsActivity.js

```diff
@@ -0,0 +1,183 @@
+import React, { Component } from 'react';
+import '../index.scss';
+import '../lists.scss';
+import '../App.scss';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router';
+import { getHoursMinutes, properCaseName } from "../stringUtils";
+import DatePickerInput from "../DatePickerInput";
+import { getPrisonDescription } from '../stringUtils';
+import moment from 'moment';
+import { Link } from "react-router-dom";
+
+class ResultsActivity extends Component {
+  displayBack () {
+    return (<div className="padding-top no-print"><Link id={`back_to_menu_link`} title="Back to selection screen link" className="link backlink" to="/whereaboutssearch" >
+      <img className="back-triangle" src="/images/BackTriangle.png" alt="" width="6" height="10"/> Back to selection screen</Link></div>);
+  }
+
+  getActivityName () {
```

Comment:
a.locationId is an integer, is there a better way to compare these?

Reply:
You could do this.props.activities.filter(a => a.locationId  === Number(this.props.activity)